### ISSUE TITLE
Add Sign-in via Google 

### ISSUE THREAD

**Issue by shubham242k** [ardovic/Open-Source-Android-Weather-App#64]
Describe your issue here.
I would like to add the login and logout  feature  via google 
### Steps that will be taken
Will integrate this app with Google sign-in and can get information about user which will help in further enhancement of app

### Would you like to work on the issue?
Tell us if you would like to work on this issue
Yes I would like to work on this

**Comment by aditya3901** [comment]
Yes I would like to work on this. I'll use Firebase Authentication and integrate Google OAuth into the app for implementing Login/SignUp into your app.

**Comment by ardovic** [comment]
Great, you're welcome to contribute to our project!

**Comment by aditya3901** [comment]
I'm willing to implement Google Authentication to your app. Regarding this I've a query :- 
You've an "Authenticate" option in the Navigation Drawer. Do you want that only authenticated users should be able to use your app OR anyone can use it and they can Sign-In if they feel like it?

**Comment by ardovic** [comment]
@aditya3901  you are welcome to implement the Google sign-in. All users should be able to use the app, for now. After we might want them to provide their own API-key for weather, since it has its limits. Yet, at current stage, there is no need to do that much. Just simple implementation of Google sign-in would be just fine.

**Comment by Niranjan029** [comment]
I am currently reading your code base but your google json file is missing in code can you please provide json file of the project.
